docs/mobile-apps/features/network-capture.md

```diff
@@ -126,12 +216,27 @@ Our network capture code depends on the fact that these classes are not obfuscat
 - com.android.volley.RequestQueue
 - com.android.volley.Response
 
-### iOS
+#### iOS
 
 - Network capture works if the app uses [NSURLSession](https://developer.apple.com/documentation/foundation/nsurlsession) or a library (like [Alamofire](https://github.com/Alamofire/Alamofire) or [AFNetworking](https://github.com/AFNetworking/AFNetworking)) that uses NSURLSession inside.
 - Calls made by [NSURLConnection](https://developer.apple.com/documentation/foundation/nsurlconnection) (deprecated by Apple), will not be captured.
 
+
 ## More Information
 
+- [Network Throttling](/mobile-apps/features/network-throttling)
 - [Sauce Labs Test Results](/test-results)
 - [Sauce Labs Insights](/insights)
+- [Visualize HAR Files With The New React Network Viewer Component](https://opensource.saucelabs.com/blog/react_network_viewer)
+
+### Viewing HAR Files
+
+You need a HAR viewer to read HAR files. Sauce Labs provides a [React Network Viewer](https://opensource.saucelabs.com/network-viewer/) HAR
+viewer for your convenience.
+
+Alternatively, some other commonly used HAR viewers include:
```

Comment:
Lets add here charlesproxy and fiddler. that is the most common HAR viewer for mobile folks